packages/webapp/pages/squads/create.tsx

```diff
@@ -75,6 +109,13 @@ function CreatePost(): ReactElement {
       onAskConfirmation(true);
     },
   });
+  const { onCreateSquad, isLoading } = useSquadCreate({
+    onSuccess: (newSquad) => {
+      const form = formToJson<CreatePostProps>(formRef.current);
+
+      return onCreatePost({ ...form, sourceId: newSquad.id });
```

Comment:
Another follow up, so let's say handle was failed, this will never fire.
What happens if user updates handle re-submits, this will just retrigger?

Reply:
Yes.